### ISSUE TITLE
RARITIES OF THE BENEFACTOR BATTLE PASS 2022

### ISSUE THREAD

**Issue by rencyjane** [ValveSoftware/Dota2-Gameplay#5834]
### Description

TREASURE CANNOT BE OPENED OR EVEN PREVIEWED

### Screenshots

![dota2_MKKafYnR65](https://user-images.githubusercontent.com/119854504/205618328-3508ee1b-77e7-4fdd-8130-cb58fc48bb87.jpg)

**Comment by juaxoo** [comment]
Yes, same.. 100s of dolars worth of item and I cannot open it. Please valve review and fix

**Comment by EPICGAMER69OP** [comment]
PLS FIX

**Comment by Pyroclasmvaan** [comment]
Valve got our money, and run out

**Comment by shaileshgarg** [comment]
![20230125223609_1](https://user-images.githubusercontent.com/26450759/214631416-ef89f76b-ec08-45bf-98df-8ccbc06fa324.jpg)

Opening now, issue can be closed.
